docker-centos/init.sh

```diff
@@ -18,7 +18,7 @@ do
     $i &
 done
 
-/usr/bin/docker-containerd-current \
+NOTIFY_SOCKET=/dev/null /usr/bin/docker-containerd-current \
```

Comment:
this is to prevent to get the `READY=1` from docker-containerd.  Doing `NOTIFY_SOCKET=/dev/null` inhibits docker-containerd-current to communicate with the runC socket (which in turn communicates with the systemd socket).